sydent/http/httpclient.py

```diff
@@ -86,7 +86,7 @@ def post_json_get_nothing(self, uri, post_json, opts):
 
         response = yield self.agent.request(
             b"POST",
-            uri.encode("ascii"),
+            uri.encode("utf8"),
```

Comment:
Should this be URL encoded?

Reply:
I _think_ Twisted does the right thing here. Anyway, this function is only called twice, once with a hardcoded URL (see `sydent/sms/openmarket.py`) and one where the hostname is derived from a user input but the path is hardcoded (see `sydent/threepid/bind.py`).

Reply:
Cool! 👍 Thanks for checking.